project/ValidatePullRequest.scala

```diff
@@ -125,25 +125,25 @@ object ValidatePullRequest extends AutoPlugin {
   def runningLocally: Boolean = !runningOnJenkins
 
   override lazy val buildSettings = Seq(
-    sourceBranch in Global in ValidatePR := {
+    ValidatePR / sourceBranch in Global := {
```

Comment:
Still 'in' here?

Reply:
Yep, for some reasons Global is special (thanks negative lookahead in regex :) )